assembl/static2/js/app/client.js

```diff
@@ -18,12 +18,30 @@ const myFragmentMatcher = new IntrospectionFragmentMatcher({
   }
 });
 
+// The object id retrieved is already unique, it's actually
+// ObjectType:primaryKey encoded in base64, so we define our
+// own dataIdFromObject instead of using the default one `${o.__typename}:o.id`.
+// This allows us to define a custom resolver for the node query.
+// for more info about customResolvers, read
+// http://dev.apollodata.com/react/query-splitting.html
```

Comment:
One more thing to learn :D So I'm not able to evaluate this commit (5d03f509be33ab03e9fe4f4936a748e4c6298533) yet.